src/celeritas/ext/VecgeomParams.cc

```diff
@@ -75,19 +75,25 @@ VecgeomParams::VecgeomParams(G4VPhysicalVolume const* world)
     CELER_EXPECT(world);
     ScopedMem record_mem("VecgeomParams.construct");
 
-    // Convert the geometry to VecGeom
-#if CELERITAS_USE_GEANT4
     {
-        ScopedMem record_mem("VecgeomParams.convert");
-        detail::GeantGeoConverter convert;
-        convert(world);
-        // save the logicalVolume ID map before converter goes out of scope
-        g4log_volid_map_ = convert.get_g4logvol_id_map();
-        CELER_ASSERT(vecgeom::GeoManager::Instance().GetWorld());
+        // Convert the geometry to VecGeom
+        g4vg::Converter::Options opts;
+        opts.compare_volumes = [] {
+            std::string var = celeritas::getenv("G4VG_COMPARE_VOLUMES");
+            return !var.empty() || var == "0";
```

Comment:
Quite so. But since none of the other environment variable conditions (e.g. `CELER_DISABLE_DEVICE`) check for the `== "0"` case I think I'll just kill it entirely.